virtcontainers/device/config/config.go

```diff
@@ -232,7 +261,9 @@ func GetHostPath(devInfo DeviceInfo) (string, error) {
 	}
 
 	format := strconv.FormatInt(devInfo.Major, 10) + ":" + strconv.FormatInt(devInfo.Minor, 10)
-	sysDevPath := filepath.Join(SysDevPrefix, pathComp, format, "uevent")
+	sysDevPath := filepath.Join(sysDevPrefix, pathComp, format, "uevent")
+
+	fmt.Errorf("sysdevpath %s", sysDevPath)
```

Comment:
and this is for ...

Reply:
Thanks for you review @devimc 
Let me add a link of drafted documentation here later.